packages/react-dom/src/client/DOMPropertyOperations.js

```diff
@@ -18,20 +18,40 @@ import {
 import sanitizeURL from '../shared/sanitizeURL';
 import {disableJavaScriptURLs} from 'shared/ReactFeatureFlags';
 
+import warning from '../shared/warnValidStyle';
+
 import type {PropertyInfo} from '../shared/DOMProperty';
 
 /**
  * Get the value for a property on a node. Only used in DEV for SSR validation.
  * The "expected" argument is used as a hint of what the expected value is.
  * Some properties have multiple equivalent values.
  */
+
+let stringifyWithPerformanceWarning;
+
 export function getValueForProperty(
   node: Element,
   name: string,
   expected: mixed,
   propertyInfo: PropertyInfo,
 ): mixed {
   if (__DEV__) {
+    stringifyWithPerformanceWarning = function(value) {
```

Comment:
Could you hoist this to a top level function? In prod, it won't have any references and will get removed, so it should be fine there.

Reply:
ok

Reply:
@threepointone Do you know why my test cases failling, i just clones and `yarn test` its failing